docs/documentation/release_notes/topics/26_1_0.adoc

```diff
@@ -43,6 +49,39 @@ ifeval::[{project_community}==true]
 Many thanks to https://github.com/sonOfRa[Simon Levermann] for the contribution.
 endif::[]
 
+= Support for prompt=create
+
+There is a support for the https://openid.net/specs/openid-connect-prompt-create-1_0.html[Initiating user registration standard], which allows OIDC client to initiate the login request with
+the parameter `prompt=create` to notify {project_name} that new user should be registered rather than existing user authenticated. Initiating user registration was already supported in {project_name} with the use of dedicated endpoint `/realms/<realm>/protocol/openid-connect/registrations`
+however this endpoint is now deprecated in favor of the standard way as it was proprietary solution specific to {project_name}.
+
+ifeval::[{project_community}==true]
+Many thanks to https://github.com/thomasdarimont[Thomas Darimont] for the contribution.
+endif::[]
+
+= Maximum count and length for additional parameters sent to OIDC authentication request
+
+The OIDC authentication request supports limited amount of additional custom parameters of maximum length. The additional parameters can be
+used for the custom purposes (EG. adding the claims into the token with the use of the protocol mappers). In the previous versions, the maximum count of
+the parameters was hardcoded to 5 and the maximum length of the parameters was hardcoded to 2000. Now both values are configurable. Additionally it can be possible to configure
+if additional parameters will cause request to fail or if parameters will be ignored.
```

Comment:
```suggestion
The OIDC authentication request supports a limited amount of additional custom parameters of maximum length. The additional parameters can be
used for the custom purposes (for example, adding the claims into the token with the use of the protocol mappers). In the previous versions, the maximum count of
the parameters was hardcoded to 5 and the maximum length of the parameters was hardcoded to 2000. Now both values are configurable. Additionally it can be possible to configure
if additional parameters will cause a request to fail or if parameters will be ignored.
```